src/client/components/Delete_Comfim/ModalList.component.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../button/button.component';
+
+const ModalList = ({ isShowing, hide }) =>
+  isShowing
```

Comment:
I suggest decomposition. At least 2-3 subcomponents

Reply:
Hi senner008, thank you for your review. Can you tell more details about what I have to do so this component be perfect?

Reply:
Well, what does it contain?